code/__defines/turfs.dm

```diff
@@ -20,4 +20,6 @@
 #define SMOOTH_ALL 1	//Smooth with all of type
 #define SMOOTH_WHITELIST 2	//Smooth with a whitelist of subtypes
 #define SMOOTH_BLACKLIST 3 //Smooth with all but a blacklist of subtypes
-#define SMOOTH_GREYLIST 4 // Use a whitelist and a blacklist at the same time. atom smoothing only
\ No newline at end of file
+#define SMOOTH_GREYLIST 4 // Use a whitelist and a blacklist at the same time. atom smoothing only
+
+#define RANGE_TURFS(CENTER, RADIUS) block(locate(max(CENTER.x-(RADIUS), 1), max(CENTER.y-(RADIUS),1), CENTER.z), locate(min(CENTER.x+(RADIUS), world.maxx), min(CENTER.y+(RADIUS), world.maxy), CENTER.z))
```

Comment:
we already have trange for this, check if this is needed at all

Reply:
This check moved to special define turfs

Reply:
I don't understand this response. Why does this define exist? is trange not good enough?

Reply:
> I don't understand this response. Why does this define exist? is trange not good enough?

I only moved check. No create new check